lib/cuke_linter/linters/unique_scenario_names_linter.rb

```diff
@@ -0,0 +1,108 @@
+module CukeLinter
+  # A linter that detects non-unique scenario names
+  class UniqueScenarioNamesLinter < Linter
```

Comment:
There is also name method, mentioned in [cuke_linter/testing/cucumber/features/custom_linters.feature at master · enkessler/cuke_linter](https://github.com/enkessler/cuke_linter/blob/master/testing/cucumber/features/custom_linters.feature)

Reply:
How could it be used?

Reply:
I've checked. there is no name in any of existing linters